src/applicants/applicant.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {Entity, Column, PrimaryGeneratedColumn} from "typeorm"
+
+@Entity()
```

Comment:
The users requesting waitlist gives us only an email and a category, they're not applicants yet. SO this should be a waitlist table.